dablooms-toolfile.spec

```diff
@@ -0,0 +1,21 @@
+### RPM external dablooms-toolfile 1.0
+Requires: dablooms
+
+%prep
+
+%build
+
+%install
+
+mkdir -p %{i}/etc/scram.d
+cat << \EOF_TOOLFILE >%{i}/etc/scram.d/dablooms.xml
+<tool name="dablooms" version="@TOOL_VERSION@">
+  <lib name="dablooms"/>
+  <client>
+    <environment name="DABLOOMS_BASE" default="@TOOL_ROOT@"/>
+    <environment name="INCLUDE" default="$DABLOOMS_BASE/include"/>
```

Comment:
@gartung , looks like you are missing LIBDIR here?

Reply:
Good catch. I must have picked it up because py2-dablooms was configured and linked libdablooms into external.

Reply:
And I didn't reconfigure externals when I did scram setup dablooms

Reply:
The library was picked up from CMSSW_RELEASE_BASE externals.